compliance_checker/cfutil.py

```diff
@@ -677,20 +677,6 @@ def get_axis_map(ds, variable):
             if coord_name not in axis_map[axis]:
                 axis_map[axis].append(coord_name)
 
-    # Sometimes dimensionless coordinates are considered coordinate variables.
```

Comment:
According to [CF](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#scalar-coordinate-variables):
> Since there is no associated dimension these scalar coordinate variables should be attached to a data variable via the coordinates attribute.

Alternatively, such a variable can be given a dimension of the same name (of size 1) and then be attached to other variables via its dimension.

Either way, the code above would find this coordinate variable. If it didn't find it, then it's not meant to be associated with the `variable` under consideration, and should _not_ be included in the `axis_map`.